README.md

```diff
@@ -53,7 +53,8 @@ Then build with the corresponding option:
  * `-DUSE_FLYCAP=true`
  * `-DUSE_V4L=true`
  
-Example: `cd build; cmake -DUSE_SPINNAKER=true ..`. As these are cached cmake option, you only need to run this once and can build with `make` afterwards.
+Example for a release build: `cd build; cmake -DUSE_SPINNAKER=true -DCMAKE_BUILD_TYPE=Release ..`.
```

Comment:
the build type is also passed to cmake in the Makefile, so specifying it here should not have any effect.

Reply:
Fixed this. We were not aware, we always ran make in the CMake directory